src/main/java/springboot/onlinebookstore/dto/book/BookWithoutCategoryIdsDto.java

```diff
@@ -0,0 +1,9 @@
+package springboot.onlinebookstore.dto.book;
+
+import java.math.BigDecimal;
+
+public record BookWithoutCategoryIdsDto(
+        Long id, String title, String author,
+        String isbn, BigDecimal price,
+        String description, String coverImage) {
```

Comment:
for code readability give the whole line for single field
```suggestion
        Long id, 
        String title, 
        String author,
        String isbn, 
        BigDecimal price,
        String description, 
        String coverImage) {
```